cg/store/crud/read.py

```diff
@@ -1430,3 +1430,37 @@ def get_order_by_ticket_id(self, ticket_id: int) -> Order | None:
             orders=orders, filters=order_filter_functions, ticket_id=ticket_id
         )
         return orders.first()
+
+    def get_not_received_case_count(self, order_id: int) -> int:
```

Comment:
I think the name is good. Something to consider is the autocomplete in the IDE is a slight reshuffle. Resolve if you think it is overkill.
```suggestion
    def get_case_not_received_count(self, order_id: int) -> int:
```
Would cut out a lot of functions not related to case queries when writing `store.get_case_` in the IDE and could finding what you need easier. Similar to order code related to tables in specific modules I guess.

Reply:
Fixed in https://github.com/Clinical-Genomics/cg/pull/3050/commits/189b4fb1393b81cb492f603e058f85ff53a82abf and https://github.com/Clinical-Genomics/cg/pull/3050/commits/bfe3f1183779272c5d79b4d151b7acdb32a91efb